Zend/tests/bug52193.phpt

```diff
@@ -35,43 +35,85 @@ array(0) {
 }
 array(1) {
   [0]=>
-  object(Closure)#%d (0) {
+  object(Closure)#%d (3) {
+    ["name"]=>
+    string(%d) "{closure:%s:%d}"
```

Comment:
Do we really need both? I would personally drop `name`.

Reply:
I just pushed another commit, because I managed to include the parent function name within the closure name, which I believe further improves the feature:

1. The closure names are even more useful, because they are more specific.
2. The memory usage will be lower for deep directory hierarchies, because the common filename prefix will not be included in the closure name.
3. The closure name will remain stable even if the application is deployed into changing directory names for atomic deployments.

I believe this also makes the `name` useful within the debug output, as it contains additional information over just the file + line.